www/scripts/codecheckerviewer/ListOfRuns.js

```diff
@@ -80,7 +80,7 @@ function (declare, dom, ItemFileWriteStore, topic, Dialog, Button,
       this.structure = [
         { name : 'Diff', field : 'diff', styles : 'text-align: center;', formatter : diffBtnFormatter},
         { name : 'Name', field : 'name', styles : 'text-align: left;', width : '100%' },
-        { name : 'Number of reports', field : 'numberofbugs', styles : 'text-align: center;', width : '20%' },
+        { name : 'Number of unfixed reports', field : 'numberofbugs', styles : 'text-align: center;', width : '20%' },
```

Comment:
Could we use `unresolved` instead of `unfixed` to be consistent with the filtering and other namings?